src/components/advanced-search/advanced-search-dates.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <v-dialog id="advanced-search-dates" v-model="dialog" persistent max-width="44rem">
+    <v-card height="32rem" class="py-6">
+      <v-row>
+        <v-col>
+          <span class="title-bold-16" :class="{ 'error-message': validate && !hasStartDate }">Select Start Date:</span>
+          <v-date-picker
+            id="start-date-calendar"
+            class="mt-2"
+            v-model="startDateText"
+            :max="currentDateString"
+            elevation="5"
+          >
+          </v-date-picker>
+        </v-col>
+        <v-col>
+          <span class="title-bold-16" :class="{ 'error-message': validate && !hasEndDate }">Select Start Date:</span>
+          <v-date-picker
+            id="end-date-calendar"
+            class="mt-2"
+            v-model="endDateText"
+            :max="currentDateString"
+            elevation="5"
+          >
+          </v-date-picker>
+        </v-col>
+      </v-row>
+      <v-card-actions class="my-0 py-0 float-right">
+          <v-btn id="btn-done" text color="primary" :ripple="false" @click="submitDates()"><strong>OK</strong></v-btn>
+          <v-btn id="btn-cancel" text color="primary" :ripple="false" @click="emitClose()">Cancel</v-btn>
```

Comment:
This is the first time I have seen the OK button bolded.
Also, in other places the Cancel button has gray styling (see upgrade.vue).

Do we want to proceed with different styles than elsewhere in this app?

Reply:
Great question. I'm not actually sure about either, this is most likely due to habit of doing so since it was requested in UX feedback prior. 
I don't believe we've standardized this 😁